Doc/library/os.path.rst

```diff
@@ -304,12 +304,13 @@ the :mod:`glob` module.)
 
 .. function:: join(path, *paths)
 
-   Join one or more path components intelligently.  The return value is the
-   concatenation of *path* and any members of *\*paths* with exactly one
-   directory separator following each non-empty part except the last, meaning
-   that the result will only end in a separator if the last part is empty.  If
-   a component is an absolute path, all previous components are thrown away
-   and joining continues from the absolute path component.
+   Join one or more path components intelligently. The return value is the
+   concatenation of path and any members of *\*paths* so that there is a
```

Comment:
`path` is a parameter name, so it should keep its `*` markings:
```suggestion
   concatenation of *path* and any members of *\*paths* so that there is a
```